SpatialGDK/Source/SpatialGDK/Private/SpatialView/ConnectionHandler/SpatialOSConnectionHandler.cpp

```diff
@@ -180,7 +182,10 @@ void SpatialOSConnectionHandler::ConnectionDeleter::operator()(Worker_Connection
 {
 	if (ConnectionToDelete != nullptr)
 	{
-		Worker_Connection_Destroy(ConnectionToDelete);
+		// todo unr-4211 - This is a mitigation for the slow connection destruction code in pie.
```

Comment:
nit: 
```suggestion
		// TODO(UNR-4211): This is a mitigation for the slow connection destruction code in pie. This will be removed when...(insert whatever event- is it when we move to v15?)
```

Reply:
I think style guide is 
```
// TODO: UNR-4211 - some text about why we need a todo here.
```

Reply:
As long as it's consistent it's good. It's just for searching for todos throughout the codebase, and being able to find them by ticket in case there are multiple for the same ticket. Although hopefully these are the only places where we're writing ticket numbers in anyway.

Reply:
Will fix it to match the guide